docs/requirements.txt

```diff
@@ -1,4 +1,5 @@
 numpy>=1.18.1
 sphinx_rtd_theme
-sphinx==3.1.0
-sphinx-click==2.3.2
\ No newline at end of file
+sphinx==3.1.2
```

Comment:
dunno if exact requirements might cause problems?  Hope this is not built when users build their code.

Reply:
No, these are only used for building the documentation. Actually it's the lack of exact versions that would have caused problems, as some environments like read-the-docs use older, incompatible sphinx by default.